go/libkb/user_card.go

```diff
@@ -22,6 +22,10 @@ type card struct {
 	TeamShowcase         []keybase1.UserTeamShowcase `json:"team_showcase"`
 	RegisteredForAirdrop bool                        `json:"airdrop_registered"`
 	Blocked              bool                        `json:"blocked"`
+	UserBlocks           struct {
```

Comment:
feels weird to me that you're putting all of *my* user blocks in an `card` corresponding to someone else's profile.

Reply:
We aren't. This is just how the blocks pertaining to that user are returned by the server. Note that it's not an array. It's just the chat and follow bits for this target user.

Reply:
You're right, these should be booleans.

Reply:
I guess I was testing with the admin service instead of dev service. Oops!